src/pages/__tests__/Settings.test.js

```diff
@@ -0,0 +1,110 @@
+import axios from 'axios'
+import { render, waitFor, fireEvent } from '@testing-library/vue'
+import Settings from '../Settings.vue'
+import wait from 'waait'
+
+jest.mock('axios')
+
+Object.defineProperty(global, 'crypto', {
+  value: {
+    subtle: {
+      digest: () => Promise.resolve('unchecked string')
+    }
+  }
+});
```

Comment:
What's this do?

Reply:
Without this, when `crypto.subtle.digest` is called on line 115 of holochainClient, the test will fail. Because the crypto global is defined in the browser, but not in the test environment.